tests/examples/test_plan.py

```diff
@@ -23,6 +23,15 @@
                            r'(?: +inventory=([\w\-.]+))?')
 
 
+def prepare_files(example, test_path, line):
+  if line is not None:
+    requested_files = line.split(',')
+    for f in requested_files:
+      destination = test_path / example.files[f].path
+      destination.parent.mkdir(parents=True, exist_ok=True)
+      destination.write_text(example.files[f].content)
+
+
 def test_example(plan_validator, tmp_path, example):
   if match := COUNT_TEST_RE.search(example.code):
```

Comment:
nit: can we use named groups in `COUNT_TEST_RE`? it makes it easier to follow the match logic. Feel free to resolve ofc, don't hold the PR for this.

Reply:
Added named groups for now. In future, I think it warrants proper parsing function that returns NamedTuple